source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -61,15 +61,31 @@ let initialize_configuration
           ~maximum_tito_depth
     |> TaintConfiguration.exception_on_error
   in
+  let () =
+    Statistics.performance
+      ~name:"Verified taint configuration"
+      ~phase_name:"Verified taint configuration"
+      ~timer
+      ()
+  in
+  taint_configuration
+
+
+let initialize_configuration_with_models ~static_analysis_configuration =
+  Log.info "Verifying model syntax.";
+  let taint_configuration = initialize_configuration ~static_analysis_configuration in
+  let timer = Timer.start () in
   (* In order to save time, sanity check models before starting the analysis. *)
   let () =
-    ModelParser.get_model_sources ~paths:taint_model_paths
+    ModelParser.get_model_sources
+      ~paths:
+        static_analysis_configuration.Configuration.StaticAnalysis.configuration.taint_model_paths
```

Comment:
Is there a reason to **not** use `taint_model_paths` here?

Reply:
I redefined the function definition  to use static_analysis_configuration as we pass that as a parameter now.  We can use the shorthand `static_analysis_configuration.configuration.taint_models_path` :)